shotover-proxy/tests/cassandra_int_tests/collections.rs

```diff
@@ -237,16 +239,31 @@ mod list {
 
         // test selecting list of sets
         for (i, native_col_type) in NATIVE_COL_TYPES.iter().enumerate() {
+            let expected: Vec<Vec<ResultValue>> = match driver {
```

Comment:
If we stick a single one of these at the top:
```rust
            let new_set = match driver {
                CassandraDriver::CdrsTokio => ResultValue::Set,
                CassandraDriver::Datastax => ResultValue::List,
            };
```

Then we can implement each of these with something like:
```rust
            assert_query_result(
                session,
                format!(
                    "SELECT my_set FROM test_collections_keyspace.test_set_table_{}_0;",
                    i
                )
                .as_str(),
                &[&[ResultValue::List(vec![new_set(vec![
                    get_type_example_result_value(*native_col_type),
                ])])]],
            )
            .await;
```